capstone/src/main/webapp/autocomplete.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Autocomplete
+ * @param {inp}
+ * @arg {arr}
+ * @return {void}
+ * Attributing this code to: w3 schools
+ * Link: https://www.w3schools.com/howto/howto_js_autocomplete.asp
+*/
+function autocomplete(inp, arr) {
+  let currentFocus;
+  inp.addEventListener('input', function(e) {
+    let a = this.value;
+    let b = this.value;
+    let i = this.value;
+    let val = this.value;
+    closeAllLists();
```

Comment:
Are you missing a parameter here? If the method signature is right and this function call is correct, it feels off to call a function that is defined to take a parameter without one. See if you can pass in an empty list instead.